docs/rackspace-infrastructure-deployment.md

```diff
@@ -0,0 +1,83 @@
+
+- **Spine switches.** Leaf switches connect to spine switches and mesh into the spine, forming the access layer that delivers network connection points for servers.
+- **Leaf switches.** Servers and storage connect to leaf  switches and consist of access switches that aggregate traffic from  servers. They connect directly to the spine.
+
+#### Advantages of Leaf-Spline Architecture
+
+- **Redundancy.** Each leaf switch connects to every spine  switch, which increases the amount of redundancy while also reducing  potential bottlenecks.
+- **Lower latency.** Spine-leaf minimizes latency and bottlenecks because each payload only has to travel to a spine switch and to another leaf switch (two hops) to reach its endpoint.
+- **Performance.** Protocols such as Shortest Path Bridging  (SPB) and Transparent Interconnection of Lots of Links (TRILL) aid in  avoiding traffic congestion.
+- **Scalability.** Additional spine switches can be added to help avoid oversubscriptionand increase scalability.
+- **Reduces spending.** A spine-leaf architecture increases  the number of connections each switch can handle, so data centers  require fewer devices to deploy.
+
+![image-20241018150739704](./assets/images/spine-leaf.png.png)
+
+
+
+#### Rackspace Network Design Details 
+
+​	Rackspace utilizes Spline-leaf network architecture where server to server traffic (east-west) has higher importance than external connectivity of the deployed application.  This is ideal for single or multi tenant deployments that process large workloads of internal data kept in **block** or **object** storage.
+
+### Commodity Storage Solutions
+
+​	Commodity storage hardware, sometimes known as off-the-shelf storage, is relatively inexpensive storage systems utilizing standard hard rives that are widely available  and basically interchangeable with other drives of its type.  These drives are housed in simple JBOD (just a bunch of disks) chassis or in smarter storage solutions such as Dell EMC or NetApp enclosures.  
+
+#### Cost effectiveness
+
+​	A major advantage of using commodity storage is for data resilience and reduced storage costs.  Because of their ability to spread data across spans of inexpensive disks, data loss risk is greatly reduced when a drive inevitably fails.  Data is automatically rebalanced to healthy drives before a degraded drive is removed from the cluster to be replaced as time permits by support staff.  
+
+#### Rackspace Storage Integrations
+
+​	Rackaspace Technology easily integrates commodity storage into its cloud solutions by leveraging it for Ceph (block/object storage) and Swift (object storage) storage targets.  
```

Comment:
Reword" Rackspace Technology easily integrates" to "Commodity storage like Ceph provide block/object etc"